tutorial/haxe/src/Main.hx

```diff
@@ -51,14 +51,44 @@ class Main {
     private static var targetPort : Int = 9090;
 
     static function main() {
-        #if ! (flash || js)
+
+        #if ! (flash || js || phpwebserver)
         try {
               ParseArgs();
         } catch (e : String) {
             trace(e);
             trace(GetHelp());
             return;
         }
+
+        #elseif  phpwebserver
```

Comment:
But we still can put everything between ```#elseif  phpwebserver``` and ```#endif``` into a separate method, couldn't we? In my opintion, we should not sacrifice code clarity to some obscure PHP edge case.